### PR TITLE
Add MongoDB 4.2 to linux images

### PR DESCRIPTION
**Pull request by dsame**
* Create installed scripts according to https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/
* Call the installed scripts from packer.io templates

The related issue https://github.com/actions/virtual-environments/issues/20

Time to install: <10 secs
Size on disk: nearly 380mb

The Ubuntu packages are not used because they install the obsolete 2.6 version of MongoDB

### CODE REVIEW

**Comment by AlenaSviridenko** [comment]
Looks like mongoDB installation scripts are the same for both ubuntu versions. If that's correct, could you please move it to common installers folder if possible? 
Or please let me know if there are any reasons why we need 2 different files.

**Comment by maxim-lobanov** [comment]
> Looks like mongoDB installation scripts are the same for both ubuntu versions. If that's correct, could you please move it to common installers folder if possible?
> Or please let me know if there are any reasons why we need 2 different files.

Looks like line 12 is different. I would suggest to add helper functions to determine OS ([like these ones](https://github.com/actions/virtual-environments/blob/master/images/win/scripts/ImageHelpers/InstallHelpers.ps1#L305)) and just add single script with branching.

**Comment by AlenaSviridenko** [comment]
@maxim-lobanov oh, I see, `ubuntu xenial` vs `ubuntu bionic`. Agree with refactoring based on OS version

File: images/linux/scripts/installers/mongodb.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+##  File:  homebrew.sh
+##  Desc:  Installs Mongo DB
+################################################################################
+
+# Source the helpers
+source $HELPER_SCRIPTS/document.sh
+
+#  Install Mongo DB
+wget -qO - https://www.mongodb.org/static/pgp/server-4.2.asc | sudo apt-key add -
+version=$(lsb_release -d|grep 'Ubuntu 16' > /dev/null  && echo xenial || echo bionic)
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/mongodb.sh]: Can we add helper function like [this one](https://github.com/actions/virtual-environments/blob/master/images/win/scripts/ImageHelpers/InstallHelpers.ps1#L305) for Ubuntu

**Comment by dsame** [comment]
build 1604 https://github.visualstudio.com/virtual-environments/_build/results?buildId=63294&view=results
build 1804 https://github.visualstudio.com/virtual-environments/_build/results?buildId=63412&view=results

File: images/linux/scripts/helpers/install.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+function TestIsXenial
+{
+    lsb_release -d|grep 'Ubuntu 16' > /dev/null
+}
+
+function TestIsBionic
+{
+    lsb_release -d|grep 'Ubuntu 18' > /dev/null
+}
```
Comment (maxim-lobanov) [inline images/linux/scripts/helpers/install.sh]: I think it is better to have the following functions:
1) `isUbuntu16` (rename from `TestIsXenial`)
2) `isUbuntu18` (rename from `TestIsBionic`)
3) `getOSVersionLabel` that will return `xenial` or `bionic`

File: images/linux/scripts/helpers/install.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+################################################################################
+##  File:  install-helpers.sh
```
Comment (sergey-akhalkov) [inline images/linux/scripts/helpers/install.sh]: nitpick: "install.sh" is not clear file name and I cannot understand what I can expect there, I would suggest to rename file to "os.sh" if it makes sense to you.

Comment (alepauly) [reply images/linux/scripts/helpers/install.sh]: @dsame I agree with @sergey-akhalkov, the script name is not clear.

**Pull request merged**